string obj
---
int16 nb_finger
int16 thumb_state
int16 distance
